cdc/model/capture.go

```diff
@@ -21,7 +21,8 @@ import (
 
 // CaptureInfo store in etcd.
 type CaptureInfo struct {
-	ID string `json:"id"`
+	ID       string `json:"id"`
+	HostName string `json:"hostname"`
```

Comment:
I think we can learn from Binlog and use `statusHost` and `statusPort` for this purpose.